pkg/rtc/room.go

```diff
@@ -472,6 +480,10 @@ func (r *Room) subscribeToExistingTracks(p types.Participant) {
 
 // broadcast an update about participant p
 func (r *Room) broadcastParticipantState(p types.Participant, skipSource bool) {
+	if p.Hidden() {
```

Comment:
I'm trying to think if there's ever a case the hidden participant should get updates about itself... I think the only thing is metadata updates.. if someone updated that participant's metadata on the server.

it seems that it's more future-proof if this function sent that through.